vyper/builtins/functions.py

```diff
@@ -1246,13 +1246,16 @@ class Send(BuiltinFunction):
 
     _id = "send"
     _inputs = [("to", AddressT()), ("value", UINT256_T)]
+    # default gas stipend is 0
+    _kwargs = {"gas": KwargSettings(UINT256_T, 0)}
```

Comment:
maybe, we should require that `gas=` is a literal. @fubuloubu ?

Reply:
i guess variable gas is a good thing since it doesn't force people to hardcode constants which have backwards compatibility issues as gas schedules change.